compaction/compaction_manager.cc

```diff
@@ -1138,12 +1140,13 @@ class compaction_manager::rewrite_sstables_compaction_task : public compaction_m
 
         while (!_sstables.empty() && can_proceed()) {
             auto sst = consume_sstable();
-            co_await rewrite_sstable(std::move(sst));
+            const auto& res = co_await rewrite_sstable(std::move(sst));
+            // FIXME: utilize compaction result
```

Comment:
What does this refer to? I see it is removed in the next commit.

Reply:
To the `res` that could be discarded in this commit. Is it not needed here?

Reply:
I think the above change is unnecessary. You can just keep the previous `co_await rewrite_sstable(std::move(sst));` as-is and only save the result into a variable in the commit in which you actually use it. It is fine to discard return-values in most cases.